mindjunkies/home/urls.py

```
@@ -5,4 +5,5 @@
 urlpatterns = [
     path("", views.home, name="home"),
     path("search/results/", views.search_view, name="search_view"),
+    
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)